### ISSUE TITLE
NVDA sometimes fails to start at the Windows login screen

### ISSUE THREAD

**Issue by harun-342** [nvaccess/nvda#18417]
### Steps to reproduce:
Configure NVDA to Start NVDA at logon, Use NVDA at logon screen (Administrator permission required), and Use current configuration during logon and secure screens (Administrator permission required). Press the power button while your computer is off.
### Actual behavior:

Windows starts, but NVDA sometimes fails to start at all, even at the Windows logon screen and even after entering the password. I get the following error message: nvda.exe - Application Error The instruction at 0x0000000076F62C34 referenced memory at 0x0000000000000005. The memory cannot be read.

Press OK to close the program
### Expected behavior:

NVDA should start up without any problems, both at the Windows login screen and afterwards, as it usually does.
### NVDA logs, crash dumps, and other attachments:

[nvda.log](https://github.com/user-attachments/files/21059063/nvda.log)

[nvda-old.log](https://github.com/user-attachments/files/21059067/nvda-old.log)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2025.1.2
#### Windows version:
Windows 11, Version 24H2 (OS Build 26100.4484)
#### Name and version of other software used to reproduce the issue:
None
#### Other information about your system:
Asus Vivobook Go E1504FA_E1504FA, AMD Ryzen 5 7520U with Radeon Graphics, 2801 Mhz, 4 Cores, 8 Logical Processors
#### Does the issue persist after restarting your computer?
Yes.
#### Have you tried other versions of NVDA? If so, please report their behavior.
This issue also applies to versions 2024.x.
#### If NVDA extensions are disabled, does the issue persist?
Yes.
#### Does the issue persist after running the COM Registry Fix Tool from the tools menu in NVDA?
Yes.

**Comment by seanbudd** [comment]
Closing as duplicate of #17453
